pkg/kubelet/metrics/metrics.go

```diff
@@ -30,30 +35,83 @@ var (
 			Help:      "Image pull latency in microseconds.",
 		},
 	)
-	// TODO(vmarmol): Implement.
-	// TODO(vmarmol): Split by source?
-	PodCount = prometheus.NewGauge(
-		prometheus.GaugeOpts{
-			Subsystem: kubeletSubsystem,
-			Name:      "pod_count",
-			Help:      "Number of pods currently running.",
-		},
-	)
-	// TODO(vmarmol): Implement.
-	// TODO(vmarmol): Split by source?
-	ContainerCount = prometheus.NewGauge(
-		prometheus.GaugeOpts{
-			Subsystem: kubeletSubsystem,
-			Name:      "container_count",
-			Help:      "Number of containers currently running.",
-		},
-	)
 	// TODO(vmarmol): Containers per pod
 	// TODO(vmarmol): Latency of pod startup
 	// TODO(vmarmol): Latency of SyncPods
 )
 
-func init() {
-	// Register the metrics.
-	prometheus.MustRegister(ImagePullLatency)
+var registerComplete bool = false
```

Comment:
Can you use sync.Once type for it?
http://golang.org/src/sync/once.go?s=260:305#L2